src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit.html.twig

```diff
@@ -156,3 +156,15 @@
     {% endif %}
   </div>
 {% endblock text_with_length_counter_widget %}
+
+{% block image_dropzone_widget %}
+  {% set attr = attr|merge({class: (attr.class|default('') ~ ' image-dropzone')|trim}) %}
+  {% set attr = attr|merge({'data-translations': translations|json_encode}) %}
```

Comment:
why repeating `{% set attr = attr|merge({` ?

Reply:
```
{% set attr = attr|merge({
     key:value,
     key: value,
     ...
}) %}
```

Reply:
I wasn't sure we could jump line in twig set But it works, you're right it's much better this way